gsoc/suborg_guidelines.md

```diff
@@ -0,0 +1,170 @@
+at the chat room of the organisation. So the student can ask the questions needed
+on the organisation room. This can also be done via e-mail, but we believe the
+students need to familiarise with the rest of the community. (Plus others can 
+help if the mentors are unavailable).
+
+Oh, and yes! you as an admin can also be a mentor. And don't forget to tell 
+them to read [Google's mentor guide].
+
+## GSOC starts
+
+If we are accepted... what happens next?
+First, have you read the [mentor guide][Google's mentor guide]? If so, take a 
+look at it again as a refresher!
+
+The application period for students is open just for two weeks. However, they
+will know for more than a month which organisations are participating. Even 
+some students start to show interest from way before that!! The main purpose
+of the programme is to bring new contributors to the organisation, so more they 
+engage better for your organisation. The dream student is that one that becomes
+mentor in the following years.
+
+### Student application
+
+Students will apply via the [GSoC portal][GSOC]. If it's not there, their
+application doesn't count. OpenAstronomy offers a template for the students to
+apply. Some suborgs require they post publicly their application on their wiki
+(see for example [SunPy's](https://github.com/sunpy/sunpy/wiki/GSoC)), others
+don't. In any case, it's good to encourage the students to share the draft of
+their applications with the mentors, so they can improve it before the deadline.
+
+OpenAstronomy has [certain rules for an application to be considered][student
+guidelines]. Familiarise with them to be able to inform your mentors and
+candidates.
+
+### Applications evaluation
+
+Each sub-org will have to evaluate the student application using a shared
+document within all the mentors. The Open Astronomy administrators will 
+share them with you. There you will grade the application, the engagement
+of the student shown so far, the quality of the pull request to the organisation,
+any notes from an interview you may do, etc.
+
+### Slots allocation
+
+Based on the numbers of mentors and outstanding applications your sub-org
+has, you will ask for a number of slots to the Open Astronomy administrators.
+They will collect these numbers from all the other sub-org and request them
+to Google.
+
+Google will then, after a few days tell us how many slots we get. Nobody 
+knows them *a priori*, and they vary considerably year after year.
+
+If the number of slots obtained is smaller than the requested then the steering
+council will decide the distribution of the slots based on the following:
+
+- The student proposal is outstanding. Project well structured and described,
+not simply a copy/paste from the ideas page.
+- What's the likelihood of that candidate student to become a long-term
+contributor? How has the student engaged till now? 
+- Mentors commitment. Mentors need to show they have committed to the organisation.
```

Comment:
```suggestion
- Commitment of the mentors for the project. The mentors need to show they have committed to the sub-org and OpenAstronomy, and evidence of this will make the slot more likely to be allocated.
```